analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Invocation.cs

```diff
@@ -113,7 +113,7 @@ protected override ProgramState[] Process(SymbolicContext context, IInvocationOp
         if (invocation.TargetMethod.IsExtensionMethod
             && invocation.TargetMethod.ReducedFrom is { } reducedFrom   // VB reduces method symbol to 'instance.Extension()' without annotated ArgumentOperation
             && reducedFrom.Parameters.First().HasNotNullAttribute()
-            && invocation.Instance.TrackedSymbol(state) is { } instanceSymbol)
+            && invocation.GetInstance(state).TrackedSymbol(state) is { } instanceSymbol)
```

Comment:
Has this worked for C# extension methods before? Please document this change in tests.

Reply:
🤦  this is an only VB block. Makes no sense.